docs/include/mkdocs-macros/main.py

```diff
@@ -14,9 +14,7 @@ def define_env(env):
     dflowfm_um_url = (
         "https://content.oss.deltares.nl/delft3dfm1d2d/D-Flow_FM_User_Manual_1D2D.pdf"
     )
-    sobek_um_url = (
-        "https://content.oss.deltares.nl/delft3d/manuals/SOBEK_User_Manual.pdf"
-    )
+    sobek_um_url = "https://content.oss.deltares.nl/sobek2/SOBEK_User_Manual.pdf"
```

Comment:
Locatie bestaat niet wanneer ik er naartoe ga met VPN aan.
SOBEK_User_Manual.pdf bestaat volgens mij in geen van de folders.

Reply:
Sinds vanmorgen 6 uur wel, ook zonder VPN. (Jobje moest nog voor de eerste keer draaien)

Reply:
yes, ik kan er bij.